Assets/Scripts/Networking/EOSTransportManager.cs

```diff
@@ -1027,29 +1043,26 @@ public void SendPacket(ProductUserId remoteUserId, string socketName, byte[] pac
             }
 
             int currentOffset = 0;
-            List<byte[]> messages = new List<byte[]>();
 
             ushort OutgoingFragmentedIndex = connection.GetNextMessageIndex();
+            byte[] fragmentBuffer = new byte[MaxPacketSize];
             for (ushort i = 0; i < numFragments; ++i)
             {
-                byte[] fragment = new byte[Mathf.Min( (packet.Length - currentOffset) + FragmentHeaderSize, MaxPacketSize)];
+                var fragment = new ArraySegment<byte>(fragmentBuffer, 0, Mathf.Min((packet.Count - currentOffset) + FragmentHeaderSize, MaxPacketSize));
+                //byte[] fragment = new byte[Mathf.Min( (packet.Length - currentOffset) + FragmentHeaderSize, MaxPacketSize)];
```

Comment:
Unless it's here for historical purposes or to illustrate something, I think this comment should be removed.

Reply:
Nope! I just missed this line when cleaning up the code. Good catch!